clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/services/LaunchTemplateService.java

```diff
@@ -297,6 +301,16 @@ private RequestLaunchTemplateData buildLaunchTemplateData(
                   .collect(Collectors.toList()));
     }
 
+    if (!asgConfig.getBlockDevicesTags().isEmpty()) {
```

Comment:
Might be preferable if this was an implementation of the `AmazonResourceTagger` interface. 

I recently added that to also support ebs volume tagging. The interface may need to change to support passing in the `AsgConfig`.

Reply:
I created a `CustomAmazonResourceTagger` class and use it in `AsgConfiguration` (AsgConfig)